### ISSUE TITLE
Libreoffice menu bar invisible

### ISSUE THREAD

**Issue by oberon2007** [horst3180/vertex-theme#187]
![libre](https://cloud.githubusercontent.com/assets/11800714/14749642/f2e7d71a-08c0-11e6-85bf-b97dacf14e83.png)

Using regular **Vertex**, commit https://github.com/horst3180/vertex-theme/commit/6c6cfdbaadb04fb2ccf3c8cf11ffcee84873af2f with **libreoffice-5.1.2** (fresh).

**Comment by oberon2007** [comment]
Working fine with "**libreoffice-still**" package, which is at 5.0.5. Might be their issue...?

**Comment by Photon89** [comment]
No, it's an issue with the GTK theme, it works well here with the Adwaita GTK3 theme (Arch Linux, GTK 3.20.3, libreoffice-fresh 5.1.2).

**Comment by fulljackz** [comment]
Hi, seems that this fix issue with Debian Stretch

`apt-get remove libreoffice-gtk3 && apt-get install libreoffice-gtk`

![capture du 2016-06-09 16-52-42](https://cloud.githubusercontent.com/assets/12140932/15934126/a1419370-2e62-11e6-8ee1-aa066f88cc36.png)

**Comment by vith** [comment]
Still an issue with
```
extra/libreoffice-fresh 5.1.4-1 [installed]
extra/gtk3 3.20.6-1 [installed]
aur/vertex-themes 20160329-2 (130, 15.41) [installed]
```
on Arch Linux.

I'm using `SAL_USE_VCLPLUGIN=gtk` as a workaround for now. This can be uncommented in `/etc/profile.d/libreoffice-fresh.sh`

**Comment by fitojb** [comment]
LibreOffice 5.2.x should fix this issue. The fixes are unfortunately too invasive to backport to 5.1.6 RC2 (the last version of the 5.1 series to be released), so please upgrade to 5.2.x.
